spec/models/pivot_identity_spec.rb

```diff
@@ -32,4 +32,21 @@
 
     it { is_expected.to eq(expected_hash) }
   end
+
+  describe "#verify_quotient_familial" do
+    subject(:verify_quotient_familial) { pivot_identity.verify_quotient_familial(quotient_familial) }
+
+    let(:quotient_familial) { build(:quotient_familial_v2_payload) }
+    let(:pivot_identity) { build(:pivot_identity, birthdate:) }
+
+    context "if my birthdate is found in the allocataires" do
+      let(:birthdate) { Date.new(1962, 8, 24) }
+      it { is_expected.to be true }
```

Comment:
une ligne entre le `let` et le `it` (on n'a pas mis rubocop rspec je suppose du coup 🤔) (pareil en dessous)

Reply:
ouais on n'a pas mis rubocop-rspec

Reply:
Tu veux l'ajouter, dans une autre PR ?